launcher-scripts/Linux-launcher

```diff
@@ -1,11 +1,5 @@
 #!/bin/bash
-
-
-if [ ! -d "jdk8u212-b03" ] 
-then
-    tar -xzf jre.tar.gz
-fi
-currentPath=`pwd`
-echo "export JAVA_HOME=$currentPath/jdk8u212-b03/" >>~/.bashrc
-echo "export PATH=$JAVA_HOME/bin:$PATH" >>~/.bashrc
-java -jar admin-console-package-1.0.0.jar
+OLD_JAVA_HOME="echo $JAVA_HOME"
```

Comment:
I could be wrong, but it feels more idiomatic if you do:

    OLD_JAVA_HOME=${JAVA_HOME}